webapp/.eslintrc.json

```diff
@@ -632,5 +632,47 @@
         "onlyEquality": false
       }
     ]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.tsx", "**/*.ts"],
+      "extends": [
+        "./node_modules/eslint-config-mattermost/.eslintrc.json",
```

Comment:
I think this shouldn't be here. We are extending the previous existing eslint config, so here we only should add the `"plugin:@typescript-eslint/recommended"` line

Reply:
removed